trino-api/README.md

```diff
@@ -0,0 +1,74 @@
+# Trino API Module
```

Comment:
merge it with main README


Let's call it as 
RZP-Trino REST client in Readme for now